tools/serverpod_cli/lib/src/generator/config.dart

```diff
@@ -6,8 +6,6 @@ import 'package:path/path.dart' as p;
 
 import 'types.dart';
 
-var config = GeneratorConfig();
```

Comment:
While this step sounds drastic, I think it's required to make serverpod_cli be usable as a package.

Reply:
👍